modules/abstract-cosmos/src/lib/utils.ts

```diff
@@ -35,7 +37,11 @@ import {
 import { CosmosKeyPair as KeyPair } from './keyPair';
 
 export class CosmosUtils implements BaseUtils {
-  private registry = new Registry([...defaultRegistryTypes]);
+  private registry;
+  constructor() {
+    this.registry = new Registry([...defaultRegistryTypes]);
+    this.registry.register('/cosmwasm.wasm.v1.MsgExecuteContract', MsgExecuteContract);
```

Comment:
nit: could be a constant

Reply:
It should already be there in constants, let me use that here